src/data/account/Account.js

```diff
@@ -43,15 +43,9 @@ export const resetPasswordAsync = async (
   confirmPassword
 ) => {
   try {
-    if (token === undefined) {
-      throw customError({
-        statusCode: 401,
-        statusText: 'Unauthorized',
-      });
-    }
     const response = await axios({
       method: 'PUT',
```

Comment:
The method here needs to be POST according to documentation.

Reply:
Are we looking at the same documentation? http://api-test.carpal.me/v3/documentation

Reply:
@maartenhemmes there are two methods, one for reset with email and use token which is PUT i think. Another one for forget password which is POST. both have same url

Reply:
yes, so resetPasswordRequestAsync is the POST and resetPasswordAsync is the PUT?